packages/stores/types.ts

```diff
@@ -1,4 +1,5 @@
-import type { GetAccountStatus, Authorize, DetailsOfEachMT5Loginid } from '@deriv/api-types';
+import moment from 'moment';
```

Comment:
```suggestion
import type { Moment } from 'moment';
```
@sergei-deriv Also you need to install `moment` as a dev dependency in `@deriv/stores`

Reply:
> @sergei-deriv Also you need to install `moment` as a dev dependency in `@deriv/stores`

Done. Thanks for your help

Reply:
@sergei-deriv please check the circlie-ci issue